homeassistant/components/zeversolar/config_flow.py

```diff
@@ -37,25 +36,12 @@ async def async_step_user(
                 step_id="user", data_schema=STEP_USER_DATA_SCHEMA
             )
 
-        errors = {}
-
-        client = zeversolar.ZeverSolarClient(host=user_input[CONF_HOST])
-        try:
-            data = await self.hass.async_add_executor_job(client.get_data)
```

Comment:
I think this is a really bad idea.

You should definitly make sure that the client is available before attempting to connect.